### ISSUE TITLE
CMOR 3.3.3 not available on pcmdi channel

### ISSUE THREAD

**Issue by durack1** [PCMDI/cmor#411]
Hi folks, it seems something is wrong with v3.3.3 not available using the following:
```
(cdat80cmor333nclnco) duro@ocean:[~]:[10567]> conda install -c pcmdi cmor
Solving environment: done
## Package Plan ##
  environment location: anaconda2/envs/cdat80cmor333nclnco
  added / updated specs:
    - cmor
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    numpy-1.13.3               |   py27hdbf6ddf_4         3.9 MB
    cmor-3.3.2.npy1.13         |   py27hb8bc26d_0         1.7 MB  pcmdi
    ------------------------------------------------------------
                                           Total:         5.5 MB
The following NEW packages will be INSTALLED:
    cmor:  3.3.2.npy1.13-py27hb8bc26d_0 pcmdi
The following packages will be DOWNGRADED:
    numpy: 1.15.0-py27h1b885b7_0              --> 1.13.3-py27hdbf6ddf_4
```

@doutriaux1 @mauzey1

**Comment by durack1** [comment]
It looks like 3.3.3 is available for numpy 1.11, 1.12, 1.13 and 1.14 (see [here](https://anaconda.org/PCMDI/cmor/files?channel=main)) so not sure what's going on

**Comment by durack1** [comment]
ROBUST!
```
(cdat80cmor333nclnco) duro@ocean:[~]:[10547]> conda install -c conda-forge cmor
Solving environment: / ^[[1;done
## Package Plan ##
  environment location: anaconda2/envs/cdat80cmor333nclnco
  added / updated specs:
    - cmor
The following packages will be downloaded:
    package                    |            build
    ---------------------------|-----------------
    cmor-3.3.3                 |   py27h18b3941_2         1.5 MB  conda-forge
The following NEW packages will be INSTALLED:
    cmor: 3.3.3-py27h18b3941_2 conda-forge
```
